contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -531,6 +534,9 @@
           this.$nextTick(this.handleValidation);
         },
       },
+      closingModal: function(newValue) {
```

Comment:
I am not sure that this guarantees that the saves will complete before the teardown occurs - we have a promise returned from the immediateSaveAll, but we're not waiting on it to resolve anywhere.

Reply:
You're right - the reason this is working okay in the preliminary manual testing I was doing is because it doesn't actually need to resolve for the updates to be made in that case. I'll think on this...